src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/GoogleDocsSubmitScreen.java

```diff
@@ -1,151 +1,108 @@
+    JFrame inputFrame = new JFrame("Google Docs Submission");
+    inputFrame.setSize(1200, 1000);
     inputFrame.setResizable(true);
     inputFrame.setVisible(true);
     return inputFrame;
   }
 
   private void addButtons() {
-    JButton addButton = newAddButton();
-    JButton removeButton = newRemoveButton();
     JButton submitButton = newSubmitButton();
 
-    JLabel buttonTitle = new JLabel("Input Controls:");
-    buttonTitle.setFont(new Font("Arial", BOLD, 14));
+    JLabel buttonTitle = new JLabel("Enter Your Google Docs Links:");
+    buttonTitle.setFont(new Font("Arial", Font.BOLD, 24));
     buttonTitle.setHorizontalAlignment(SwingConstants.CENTER);
 
     JPanel buttonPanel = new JPanel();
-    buttonPanel.add(addButton);
-    buttonPanel.add(removeButton);
     buttonPanel.add(submitButton);
+    buttonPanel.setOpaque(false);
 
-    inputPanel.add(buttonTitle);
-    inputPanel.add(buttonPanel);
+    GridBagConstraints gbc = new GridBagConstraints();
+    gbc.gridx = 0;
+    gbc.gridy = 0;
+    gbc.insets = new Insets(10, 0, 10, 0);
+    gbc.anchor = GridBagConstraints.CENTER;
+    inputPanel.add(buttonTitle, gbc);
+
+    gbc.gridy = 1;
+    inputPanel.add(buttonPanel, gbc);
   }
 
   private JButton newSubmitButton() {
-    JButton submitButton = new JButton("Submit");
+    JButton submitButton = new JButton("Verify");
+    submitButton.setPreferredSize(new Dimension(200, 50));
+    submitButton.setFont(new Font("Arial", Font.BOLD, 18));
+    submitButton.setFocusPainted(false);
     submitButton.addActionListener(e -> loadDataInBackground());
     return submitButton;
   }
 
-  private JButton newRemoveButton() {
-    JButton removeButton = new JButton("Remove Input");
-    removeButton.addActionListener(
-        e -> {
-          if (inputFields.size() > MIN_INPUTS) {
-            removeInputField();
-          } else {
-            showMessageDialog(inputFrame, "At least 1 input is required.");
-          }
-        });
-    return removeButton;
-  }
-
-  private JButton newAddButton() {
-    JButton addButton = new JButton("Add Input");
-    addButton.addActionListener(
-        e -> {
-          if (inputFields.size() < MAX_INPUTS) {
-            addInputField();
-          } else {
-            showMessageDialog(inputFrame, "Maximum of 9 inputs allowed.");
-          }
-        });
-    return addButton;
-  }
-
   private void addInitialInput() {
-    JTextField initialField = newGoogleDocsLinkTextField();
+    JTextArea initialField = newGoogleDocsLinkTextArea();
     inputFields.add(initialField);
-    inputPanel.add(initialField);
-  }
 
-  private JTextField newGoogleDocsLinkTextField() {
-    var textField = new JTextField(20);
-    textField.setInputVerifier(linkIdInputVerifier);
-    return textField;
-  }
+    GridBagConstraints gbc = new GridBagConstraints();
+    gbc.gridx = 0;
+    gbc.gridy = 2;
+    gbc.fill = GridBagConstraints.HORIZONTAL;
+    gbc.insets = new Insets(10, 50, 10, 50);
```

Comment:
use setters if possible

Reply:
In this case, GridBagConstraints is a simple Java class where the fields like gridx, gridy, fill, and insets are public and directly accessible. The class does not provide setter methods for these fields because it was designed to allow direct manipulation of the fields.